### DISCUSSION TITLE
Editing customers, projects and activities throws a 500 error after update to 1.19

### DISCUSSION THREAD

**Discussion by VOOM108** [kimai/kimai#3217] [On-Premise: Installation & Updates]
After the update from 1.15 to 1.19 (PHP is now version 8.0) everything works so far, even a plugin, EasyBackup, which previously did not work for me, now works.

However, I just noticed that **editing customers, projects and activities throws a 500 error**. 

Now I know that Kimai usually has something to do with permissions, but I have otherwise adjusted the permissions correctly and **everything else works**.

Maybe someone has a hint where I could specifically look to solve this.

**Comment by VOOM108** [comment]
Solved; was a faulty plugin that would have included sth on those forms. Deleted it and everything is fine now :)

**Reply by kevinpapst** [reply]
A Kimai or browser plugin?

**Reply by VOOM108** [reply]
Kimai Plugin, the one for custom rounding per customer / project / activity.... It may have been an old version that is now incompatible, but I did not use it anymore anyway (because I would need the start and end times adapted as well, not only the resulting time...)

**Comment by VOOM108** [comment]
I still have a problem, but a very significant one... Before I deleted the plugin (reloaded the cache and fixed permissions afterwards) starting a timesheet entry worked. Now this is the only thing, that throws a 500 error: starting, restarting etc. a timesheet entry. Could this still be connected to the plugin? What else could be going on?

**Reply by kevinpapst** [reply]
I don't know, ask your log file.

**Reply by VOOM108** [reply]
Just at it... Yes, it is the plugin that left a trace in a file. Will try to replace that with the original from the repository...

**Reply by VOOM108** [reply]
That did not help, the line 145 is only where the errormessage is produced: 

`[2022-03-23 15:08:13] request.INFO: Matched route "restart_timesheet". {"route":"restart_timesheet","route_parameters":{"_route":"restart_timesheet","_controller":"App\\API\\TimesheetController::restartAction","_format":"json","id":"474"},"request_uri":"https://time.______.com/api/timesheets/474/restart","method":"PATCH"} []
[2022-03-23 15:08:13] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unknown rounding mode: glorpen.default" at /var/www/vhosts/_______.de/time._______.com/kimai2/src/Timesheet/RoundingService.php line 145 {"exception":"[object] (InvalidArgumentException(code: 0): Unknown rounding mode: glorpen.default at /var/www/vhosts/________.de/time.108design.com/kimai2/src/Timesheet/RoundingService.php:145)"} []`

Now I am stuck: where can I remove that rounding mode? In the database?

**Reply by kevinpapst** [reply]
System > Settings > Rounding

**Comment by VOOM108** [comment]
So there is this entry in the DB in kimai2_configuration: 

`timesheet.rounding.default.mode | glorpen.default`

I am not sure if I can delete that completely, or if I need another timesheet.rounding.default.mode

**Reply by VOOM108** [reply]
Simpler than thought: just had to set the default mode in the settings to sth else and then back to "ceil" which is, what is now in the database for the default mode :)

**Comment by VOOM108** [comment]
I like the new weekly worktime page etc. :) Always struggling with the updates, but less each time or at least with new adventures - and in the end happy to have everything working (again) :D
